go/client/cmd_pgp_gen.go

```diff
@@ -68,6 +73,32 @@ func (v *CmdPGPGen) Run() (err error) {
 	return err
 }
 
+// Duplicates the test in libkb.PGPGenArg.CreatePGPIDs() for interactive input
+var CheckPGPID = libkb.Checker{
+	F: func(s string) bool {
+		if !strings.Contains(s, "<") && libkb.CheckEmail.F(s) {
+			return true
+		}
+		_, err := libkb.ParseIdentity(s)
+		return err == nil
+	},
+	Hint: "2-12 letter id or email, or pgp style id",
+}
+
+func (v *CmdPGPGen) propmptPGPIDs() (err error) {
+	prompt := "Enter default ID"
```

Comment:
right - we should avoid PGP terms people won't know, like "ID". If they're an expert and want to go do their own thing, they can use GPG. Our goal is to have sensible defaults and ask them plain English questions, such as what email addresses they're known by, and what their name is.